q2cli/tools.py

```diff
@@ -37,14 +37,36 @@ def export_data(path, output_dir):
     result.export_data(output_dir)
 
 
+def show_importable_types(ctx, param, value):
+
```

Comment:
@jairideout Roger on both comments. 

Would you like to `self.assertTrue('IntSequence1' in result.output)` for all expected outputs, as I did in my commit, or is that overkill?  If not, how many exactly do you want to test for?